modules/apps/document-library/document-library-item-selector-web/src/main/java/com/liferay/document/library/item/selector/web/internal/display/context/DLItemSelectorViewDisplayContext.java

```diff
@@ -400,7 +400,7 @@ private long _getFileEntryTypeId(String fileEntryTypeKey) {
 			return dlFileEntryType.getFileEntryTypeId();
 		}
 
-		return 0;
+		return null;
```

Comment:
Returning `null` is better 👍  but you missed handling it in the calling method, where the returned value is assigned to a `long` primitive data type

Reply:
Done